lib/command_duration.bash

```diff
@@ -59,16 +59,16 @@ function _command_duration() {
 		command_duration=0
 	fi
 
-	if ((command_duration > 0)); then
+	if ((command_duration >= COMMAND_DURATION_MIN_SECONDS)); then
 		minutes=$((command_duration / 60))
 		seconds=$((command_duration % 60))
-	fi
 
-	_dynamic_clock_icon "${command_duration}"
-	if ((minutes > 0)); then
-		printf "%s %s%dm %ds" "${COMMAND_DURATION_ICON:-}" "${COMMAND_DURATION_COLOR:-}" "$minutes" "$seconds"
-	elif ((seconds >= COMMAND_DURATION_MIN_SECONDS)); then
-		printf "%s %s%d.%01ds" "${COMMAND_DURATION_ICON:-}" "${COMMAND_DURATION_COLOR:-}" "$seconds" "$deciseconds"
+		_dynamic_clock_icon "${command_duration}"
+		if ((minutes > 0)) && ((command_duration >= COMMAND_DURATION_MIN_SECONDS)); then
+			printf "%s %s%dm %ds" "${COMMAND_DURATION_ICON:-}" "${COMMAND_DURATION_COLOR:-}" "$minutes" "$seconds"
+		elif ((seconds >= COMMAND_DURATION_MIN_SECONDS)); then
+			printf "%s %s%d.%01ds" "${COMMAND_DURATION_ICON:-}" "${COMMAND_DURATION_COLOR:-}" "$seconds" "$deciseconds"
```

Comment:
Don't need the `COMMAND_DURATION_MIN_SECONDS` checks in here since we take care of it in the outer loop now ...

Reply:
Good catch, my bad, got ahead of myself.